### ISSUE TITLE
The performance of default voc8.res50v3+.CPS

### ISSUE THREAD

**Issue by swt199211** [charlesCXK/TorchSemiSeg#39]
Hello, I'm training with double 3080 cards. The batchsize can only be set to 4. The learning rate is 0.0025. The ratio of 1 / 8. I've run it three times and the IOU is only 0.66. What's the reason?Does batchsize have a great impact?

**Comment by XieZhiyao21** [comment]
Have you solved your problem?What changes do you make?
